### DISCUSSION TITLE
How do you get dA/dt in a TMS simulation

### DISCUSSION THREAD

**Discussion by rfparra** [simnibs/simnibs#323] [8. Simulation workflows]
Dear Colleagues,

I am trying to calculate the value of B and E fields by myself with custom coils to compare them with the values that the GUI gives me. The problem is that I don't know how to calculate these fields, the solutions within the script "analytical_solutions.py" don't help me because I only know the coil dimensions and its di/dt value.

So how do you calculate these fields or in which script do you do it. Also, I don't know how to get the dA/dt values having only the dimensions of a coil and its di/dt.

Many thanks in advance for your help.

Best regards,
Rafael.

**Comment by merdi-dtu** [comment]
We are currently working on coil definitions based on line integrals. Once this code is ready, it will probably be useful for you to see how to calculate the E-field. Please check for this again in a few months.
